hms-app/src/common/styles.ts

```diff
@@ -46,6 +46,12 @@ export const styles = createStyles((theme) => ({
     lineHeight: '1.5em',
   },
 
+  smallText: {
```

Comment:
I feel like there is some way to extend another style... I would think this should just extend `text` and overwrite the properties which should be different

Reply:
not sure if this would not overcommplicate it

Reply:
the point is more about being able to keep things consistent easily when you change them.  And if the amount of things you set in each of these becomes large then it does make for less code later